src/App.js

```diff
@@ -1,6 +1,8 @@
+import Calculator from './components/Calculator';
+
 const App = () => (
-  <div className="App">
-    <h1>Hello World</h1>
+  <div>
+    <Calculator />
   </div>
 );
 
```

Comment:
### Although I am highlighting this file, the following change should be made in the `public` directory 

- [ ]  One of the requirements is to delete all files that were created with the create react command that won't be relevant to this project. In light of this, kindly delete the following files in the _**public**_ directory 👍 😎:

- In the `public` directory:
  - favicon.ico
  - logo192.png
  - logo512.png
  - manifest.json
  - robots.txt

Reply:
done.

Reply:
- [ ] This has not been resolved as the previous reviewer mentioned, kindly delete all the files in the public directory listed by the previous user :+1: